gems/aws-sdk-dynamodbstreams/lib/aws-sdk-dynamodbstreams/client.rb

```diff
@@ -266,6 +268,14 @@ class Client < Seahorse::Client::Base
     #
     #   @option options [String] :session_token
     #
+    #   @option options [Boolean] :simple_attributes (false)
+    #     Enables working with DynamoDB attribute values using
+    #     hashes, arrays, sets, integers, floats, booleans, and nil.
+    #
+    #     Disabling this option requires that all attribute values have
```

Comment:
This really only applies to inputs.

Reply:
Yeah, I fixed the docs but didn't rebuild.